README.md

```diff
@@ -49,23 +49,37 @@ You can build your precompile and Subnet-EVM with `./scripts/build.sh`. This scr
 
 ### Run
 
-You can run you Precompile-EVM by using Avalanche Network Runner
+You can run you Precompile-EVM by using the Avalanche CLI.
+
+First, create the configuration for your subnet.
 
 ```bash
-avalanche-network-runner server \
---log-level debug \
---port=":8080" \
---grpc-gateway-port=":8081"
+avalanche subnet create mysubnet
```

Comment:
Please also check if there is a one-line command that where users don't have to go through the dialogue and offer as alternative for advanced users. We should state both options